stdlib/influxdata/influxdb/monitor/state_changes_custom_any_to_any_test.flux

```diff
@@ -0,0 +1,54 @@
+package monitor_test
+
+import "influxdata/influxdb/monitor"
+import "influxdata/influxdb/v1"
+import "testing"
+import "experimental"
+
+option now = () => 2018-05-22T19:54:40Z
+
+option monitor.log = (tables=<-) => tables |> drop(columns:["_start", "_stop"])
+
+// These statuses were produce by custom check query
+inData = "
```

Comment:
@jlapacik pointed out that we have to provide tables in an unpivotted format so that these tests will pass when ran against influxdb. Could you switch this to the unpivotted format? Just make the same call you did to get this data, but don't add the `v1.fieldsAsCols()` call.